activation/poetlistener_test.go

```diff
@@ -34,6 +36,7 @@ func (ServiceMock) Broadcast(protocol string, payload []byte) error { panic("imp
 func (ServiceMock) Shutdown() { panic("implement me") }
 
 type mockMsg struct {
```

Comment:
Nothing special here. The usual tricky stuff about using structs in the same package scope, so it's tempting to access internal vars that aren't guarded.